examples/gno.land/p/demo/ownable/ownable_test.gno

```diff
@@ -6,62 +6,62 @@ import (
 )
 
 var (
-	firstCaller  = std.Address("g1l9aypkr8xfvs82zeux486ddzec88ty69lue9de")
-	secondCaller = std.Address("g127jydsh6cms3lrtdenydxsckh23a8d6emqcvfa")
+	firstAddr  = std.Address("g1wymu47drhr0kuq2098m792lytgtj2nyx77yrsm")
```

Comment:
This address is the default context one from tests. By using this address here, you're concealing a bug with `std.TestSetRealm`.

Reply:
i found that the unit test doesnt work correctly with the old address , so i changed it

Reply:
https://github.com/gnolang/gno/pull/2198/commits/8bd3ac0bd411c9f2adee3bbc466e57ed6f33ac5b